src/DETHRACE/common/car.c

```diff
@@ -4213,7 +4137,7 @@ void MungeCarGraphics(tU32 pFrame_period) {
                     if (the_car->gear) {
                         wheel_speed = -(the_car->revs
                             * the_car->speed_revs_ratio
-                            / 6900.0
+                            / (WORLD_SCALE * 1000.f)
```

Comment:
Is this the max rpm, not related to world scale?

Reply:
You're probably right, I assumed it is the conversion factor from rpm (1/s) to speed (m/s) which needs to know the world scale.
But still, the number is suspicious.
```suggestion
                            / (6900.f * 1000.f)
```